app/javascript/components/miq-ae-class/index.jsx

```diff
@@ -0,0 +1,193 @@
+import React, { useState, useEffect } from 'react';
+import { FormSpy } from '@data-driven-forms/react-form-renderer';
+import { Button } from 'carbon-components-react';
+import MiqFormRenderer, { useFormApi } from '@@ddf';
+import PropTypes from 'prop-types';
+import createSchema from './class-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+import miqFlash from '../../helpers/miq-flash';
+
+const MiqAeClass = ({ classRecord, fqname }) => {
+  const formattedFqname = fqname.replace(/\s+/g, '');
```

Comment:
Is this implying that the fqname comes back from the API with spaces?

Reply:
I was going through the MiqAeClassController and saw [this line of code](https://github.com/ManageIQ/manageiq-ui-classic/blob/88695dc56d6eb7f9259146662be8585e111d608e/app/controllers/miq_ae_class_controller.rb#L134) which is creating those spaces. 

Not sure if this line was added intentionally. If it's removed, it may affect the Fully Qualified Name displayed during the creation/editing of Domain, Namespace, Class (and potentially other areas). Could you please confirm if it's fine to remove it?

Reply:
Yes, we can remove that.